server/events/policy_check_project_command_builder.go

```diff
@@ -0,0 +1,89 @@
+package events
+
+import (
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/events/vcs"
+	"github.com/runatlantis/atlantis/server/events/yaml"
+	"github.com/runatlantis/atlantis/server/events/yaml/valid"
+)
+
+func NewPolicyCheckProjectCommandBuilder(p *DefaultProjectCommandBuilder) *PolicyCheckProjectCommandBuilder {
+	return &PolicyCheckProjectCommandBuilder{
+		ProjectCommandBuilder: p,
+		ParserValidator:       p.ParserValidator,
+		ProjectFinder:         p.ProjectFinder,
+		VCSClient:             p.VCSClient,
+		WorkingDir:            p.WorkingDir,
+		WorkingDirLocker:      p.WorkingDirLocker,
+		CommentBuilder:        p.CommentBuilder,
+		GlobalCfg:             p.GlobalCfg,
+	}
+}
+
+type PolicyCheckProjectCommandBuilder struct {
+	ProjectCommandBuilder *DefaultProjectCommandBuilder
+	ParserValidator       *yaml.ParserValidator
+	ProjectFinder         ProjectFinder
+	VCSClient             vcs.Client
+	WorkingDir            WorkingDir
+	WorkingDirLocker      WorkingDirLocker
+	GlobalCfg             valid.GlobalCfg
+	PendingPlanFinder     *DefaultPendingPlanFinder
+	CommentBuilder        CommentBuilder
+	SkipCloneNoChanges    bool
+}
+
+func (p *PolicyCheckProjectCommandBuilder) BuildAutoplanCommands(ctx *CommandContext) ([]models.ProjectCommandContext, error) {
+	projectCmds, err := p.ProjectCommandBuilder.BuildAutoplanCommands(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	commentCmd := &CommentCommand{
+		Verbose: false,
+		Flags:   nil,
+	}
```

Comment:
Should we not respect verbosity for policy checking as well?

Reply:
Autoplan doesn't do verbosity. If you look in `BuildAutoplanCommands` in the DefaultProjectCommandBuilder. But we do support it for comments. If `atlantis plan` is setup to be verbose policy_check will inherit it since I'm using the same comment command.